src/pool/srv_metrics.c

```diff
@@ -42,21 +42,44 @@ ds_pool_metrics_alloc(const char *path, int tgt_id)
 	if (metrics == NULL)
 		return NULL;
 
-	rc = d_tm_add_metric(&metrics->open_hdl_gauge, D_TM_GAUGE,
-			     "Number of open pool handles", "hdls",
-			     "%s/pool_handles", path);
-	if (rc != 0)
-		D_ERROR("Couldn't add open handle gauge: "DF_RC"\n", DP_RC(rc));
-
 	rc = d_tm_add_metric(&started, D_TM_TIMESTAMP,
 			     "Last time the pool started", NULL,
 			     "%s/started_at", path);
 	if (rc != 0) /* Probably a bad sign, but not fatal */
-		D_ERROR("Failed to add started_timestamp metric, " DF_RC "\n",
-			 DP_RC(rc));
+		D_WARN("Failed to create started_timestamp metric: "DF_RC"\n", DP_RC(rc));
 	else
 		d_tm_record_timestamp(started);
 
+	rc = d_tm_add_metric(&metrics->evict_total, D_TM_COUNTER,
+			     "Total number of pool handle evict operations", "ops",
+			     "%s/ops/pool_evict", path);
+	if (rc != 0)
+		D_WARN("Failed to create evict hdl counter: "DF_RC"\n", DP_RC(rc));
+
+	rc = d_tm_add_metric(&metrics->connect_total, D_TM_COUNTER,
+			     "Total number of processed pool connect operations", "ops",
+			     "%s/ops/pool_connect", path);
+	if (rc != 0)
+		D_WARN("Failed to create pool connect counter: "DF_RC"\n", DP_RC(rc));
+
+	rc = d_tm_add_metric(&metrics->disconnect_total, D_TM_COUNTER,
+			     "Total number of processed pool disconnect operations", "ops",
+			     "%s/ops/pool_disconnect", path);
+	if (rc != 0)
+		D_WARN("Failed to create pool disconnect counter: "DF_RC"\n", DP_RC(rc));
+
+	rc = d_tm_add_metric(&metrics->query_total, D_TM_COUNTER,
+			     "Total number of processed pool query operations", "ops",
+			     "%s/ops/pool_query", path);
```

Comment:
Including with space (as following) case or not?

Reply:
Yes, including space. Fixed.